packages/react-instantsearch-native/src/index.ts

```diff
@@ -6,6 +6,7 @@ export { translatable } from 'react-instantsearch-core';
 // Widget
 export { Configure } from 'react-instantsearch-core';
 export { QueryRuleContext } from 'react-instantsearch-core';
+export { Index } from 'react-instantsearch-core';
```

Comment:
We have to export the widget from `react-instantsearch-core` otherwise can't use the `native` package.

```js
import { InstantSearch } from 'react-instantsearch-native';

// <InstantSearch /> is not exported anymore
```